app/components/SavedContracts/SavedContracts.scss

```diff
@@ -2,7 +2,15 @@
 .saved-contracts-container {
 
   table tbody td.buttons {
-    .button { height: 29px; }
+    .button {
+      height: 29px;
+      &[aria-disabled="true"] {
+        background-color: $button-grey-disabled;
+        color: white;
+        cursor: not-allowed;
+        border: 1px solid $button-grey-disabled;
```

Comment:
I prefer opacity over dark gray.

![image](https://user-images.githubusercontent.com/6963212/41818396-df0041c6-77b6-11e8-960b-d0346f771486.png)

@elanperach what do you think? The above pic has two versions of disabled button style, and the bottom two are not disabled.

Reply:
opacity is great but I would probably remove the disabled aria value (if the button can still be clicked) as your other git diff suggests.